cmd/geth/snapshot.go

```diff
@@ -360,7 +361,11 @@ func pruneBlock(ctx *cli.Context) error {
 	if path == "" {
 		return errors.New("prune failed, did not specify the AncientPath")
 	}
-	newAncientPath = filepath.Join(path, "ancient_back")
+	if !strings.HasSuffix(oldAncientPath, "geth/chaindata/ancient/chain") {
```

Comment:
if a node old ancient path is not empty
when it use new version of geth, it will remain old path

so this modify will break it

Reply:
the old ancient path is not compatible wiith v1.3.0 after bigmerge.